modules/ROOT/pages/install-patches.adoc

```diff
@@ -0,0 +1,111 @@
+= Perform Scheduled Maintenance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the following procedure when performing scheduled maintenance for Anypoint Runtime Fabric nodes, such as applying OS patches and security updates. 
+
+== Controller Node Maintenance
+
+The controller nodes are the entry points into your Runtime Fabric cluster, but do not run any application workloads.
+
+To update controller nodes:
+
+. On your upstream TCP load balancer, remove the controller nodes from your load balancing pool, one at a time, during patching and rebooting tasks. 
+. When updates are complete on each controller node, add the node back to the cluster before removing the next one.
+
+== Worker Node Maintenance
+
+To update worker nodes: 
+
+. On a controller node, run the `gravity shell` command.
+. Verify that the status for all nodes is `READY`:
++
+```
+$ sudo kubectl get nodes
+
+NAME          STATUS   ROLES    AGE   VERSION
+172.31.0.9    Ready    node     1d   v1.x.x
+172.31.0.8    Ready    master   1d   v1.x.x
+```
+. Retrieve the application namespace (environment ID): 
++
+```
+$ sudo kubectl get ns
+```
++
+In the command output, the application namespace is represented by a long string of characters. In the following example, the application namespace
+is `fff5df7b-c49d-48c9-967d-0071412722x4`.
++
+More than one application namespace value is displayed if the RTF cluster is associated with more than 1 environment.
```

Comment:
Do we need to highlight the application namespace below for clarity?

Reply:
It's mentioned in the previous statement - happy to move it if there's a better location.

Reply:
Perhaps move the command output right below the command `kubectl get ns`, and then say _In the command output above.._ ?

Reply:
Updated this - good point!